### ISSUE TITLE
Orange SVM performs worse than sklearn SVM on identical datasets 

### ISSUE THREAD

**Issue by tastyminerals** [biolab/orange3#2530]
I have sklearn SVM implementation:
```python2
from sklearn.svm import SVC

clf = SVC()

trainset = load_data('train')
testset = load_data('test')

# train
feats_train = trainset[trainset.columns[0:4]]
targets_train = trainset['target']
clf.fit(feats_train, targets_train)
# test
feats_test = testset[testset.columns[0:4]]
targets_test = testset['target']
preds = clf.predict(feats_test)

truep, truen, falsep, falsen = 0, 0, 0, 0
for p, t in zip(preds, targets_test):
    # true positives
    if t == p and t == 1:
        truep += 1
    # true negatives
    if t == p and t == 0:
        truen += 1
    # false positives
    if t != p and t == 0:
        falsep += 1
    # false negatives
    if t != p and t == 1:
        falsen += 1

accuracy = (truep + truen) / (truep + falsep + falsen + truen)
prec = truep / (truep + falsep)
recall = truep / (truep + falsen)
f1 = 2 * (recall * prec) / (recall + prec)

```
Both **trainset** and **testset** are tab delimited files with the following stucture:

```csv
feat1 feat2 feat3 feat4 target
42.23 340 12 323 0
415 23331 34 123 0
23.1 23.11 10 8.5 1
231 100 200 100 0
```
The above script reaches **f1=0.92, prec=0.98, recall=0.87** on test data while Orange implementation struggles really hard with the exact same data.

![2017-08-16-212708_952x548_scrot](https://user-images.githubusercontent.com/7676160/29381544-d188fb74-82c9-11e7-9308-adfeb02b9883.png)

The displayed values are too low for the dataset so I wonder if there is something wrong with current svm implementation in Orange?

##### Orange version
**3.5.0.dev0+19e00c0**

**Comment by kernc** [comment]
Orange uses sklearn's SVM underneath. What differs is the preprocessing pipeline with any discrete columns first being continuized (replaced with one-hot indicator columns) and all columns normalized for SD=1. Additionally, any changes to parameters you made in the SVM dialog may play a role.

**Comment by tastyminerals** [comment]
@kernc take a look at the input table, there are no any discrete values there, 4 columns of floats.

**Comment by ajdapretnar** [comment]
As mentioned Orange uses SVM. The issue could be normalization. I have test both on breast-cancer-wisconsin.tab in the code and they work the same.
